src/Page/Object.elm

```diff
@@ -502,69 +535,127 @@ itemListView db selectedImpact results query =
+                                        -- swith to components ids as soon as they are implemented
+                                        UpdateItem
+                                            { name = name
+                                            , quantity = Query.quantity nonNullInt
+                                            , processes = processes |> List.map (\( amount, process ) -> { amount = amount, processId = process.id })
+                                            }
+                                    )
+                                |> Maybe.withDefault NoOp
                     ]
                     []
-                , span
-                    [ class "input-group-text justify-content-center fs-8"
-                    , style "width" "38px"
+                ]
+            ]
+        , td [ class "align-middle text-truncate w-100" ]
+            [ text name ]
+        , td [ class "text-end align-middle text-nowrap" ]
+            [ Format.kg <| Simulator.extractMass itemResults ]
+        , td [ class "text-end align-middle text-nowrap" ]
+            [ Simulator.extractImpacts itemResults
+                |> Format.formatImpact selectedImpact
+            ]
+        , td [ class "pe-3 align-middle text-nowrap" ]
+            [ button [ class "btn btn-outline-secondary", onClick (RemoveComponent name) ]
+                [ Icon.trash ]
+            ]
+        ]
+    , tr []
+        [ td [ colspan 5 ]
+            [ details [ class "mb-2" ]
+                [ summary [ class "ps-3" ] [ text "Procédés" ]
+                , div [ class "table-responsive" ]
+                    [ table [ class "table mb-0" ]
+                        [ thead []
+                            (tr [ class "fs-7 text-muted" ]
+                                [ th [ class "ps-3", scope "col" ] [ text "Quantité" ]
+                                , th [ scope "col" ] [ text "Procédé" ]
+                                , th [ scope "col" ] [ text "Densité" ]
+                                , th [ scope "col" ] [ text "Masse" ]
+                                , th [ scope "col" ] [ text "Impact" ]
+                                , th [ scope "col" ] [ text "" ]
+                                ]
+                                :: List.map2 (processView selectedImpact) processes (Simulator.extractItems itemResults)
+                            )
+                        ]
                     ]
-                    [ text process.unit ]
                 ]
             ]
+        ]
+    ]
+
+
+processView : Definition -> ( Query.Amount, Process ) -> Results -> Html Msg
+processView selectedImpact ( amount, process ) itemResults =
+    let
+        floatAmount =
+            amount |> Query.amountToFloat
+    in
+    tr []
+        [ td [ class "ps-3 align-middle text-nowrap" ]
+            [ case process.unit of
+                "kg" ->
+                    Format.kg (floatAmount |> Mass.kilograms)
```

Comment:
In `floatAmount |> Mass.kilograms`, the pipe operator is unnecessary. This statement should either be:

```elm
floatAmount |> Mass.kilograms |> Format.kg
```

or

```elm
Format.kg (Mass.kilograms floatAmount)
```